src/common/services/checkoutHelpers/checkout.service.spec.js

```diff
@@ -0,0 +1,55 @@
+import angular from 'angular'
+import 'angular-mocks'
+
+import module from './checkout.service'
+
+describe('checkout service', () => {
+  beforeEach(angular.mock.module(module.name))
+
+  beforeEach(angular.mock.module(($provide) => {
+    $provide.value('envService', {
+      read: () => '123'
+    })
+  }))
+
+  const self = {}
+  let script
+
+  beforeEach(inject((checkoutService, envService, $window) => {
+    self.checkoutService = checkoutService
+    self.envService = envService
+    self.$window = $window
+    self.$window.document = document
+
+    script = self.$window.document.createElement('script')
+  }))
+
+  describe('initializeRecaptcha()', () => {
+    beforeEach(() => {
+      script.src = 'https://www.google.com/recaptcha/enterprise.js?render=123'
+      script.id = 'test-script'
+    })
+
+    afterEach(() => {
+      const foundScript = self.$window.document.getElementById('give-checkout-recaptcha')
+      if (foundScript) {
+        document.head.removeChild(foundScript)
+      }
+    })
+
+    it('should add a script even if one already exists', () => {
+      self.$window.document.head.appendChild(script)
+      self.checkoutService.initializeRecaptcha.call(self)
```

Comment:
And in the following test

```suggestion
      self.checkoutService.initializeRecaptcha()
```